refactron/cicd/gitlab_ci.py

```
@@ -0,0 +1,189 @@
+            artifacts_enabled: Save analysis reports as artifacts
+
+        Returns:
+            YAML pipeline content
+        """
+        if python_versions is None:
+            python_versions = ["3.8", "3.9", "3.10", "3.11", "3.12"]
+
+        if quality_gate is None:
+            quality_gate = {"fail_on_critical": True, "max_critical": 0}
+
+        python_matrix = ", ".join([f'"{v}"' for v in python_versions])
```

Ruff (0.14.13)

Local variable `python_matrix` is assigned to but never used

Remove assignment to unused variable `python_matrix`

(F841)